<launch>
    <arg name="language" default="fr"/>

    <node pkg="smart_speaker" type="smart_speaker_node" name="smart_speaker_node" output="screen">
        <param name="language" value="$(arg language)"/>
        <param name="story_path_en" value="$(find smart_speaker)/story_en.txt"/>
        <param name="story_path_fr" value="$(find smart_speaker)/story_fr.txt"/>
        <param name="song_path" value="$(find smart_speaker)/admiralbob_wanderer.mp3"/>
    </node>

    <include file="$(find t_top)/launch/bringup.launch">
        <arg name="language" value="$(arg language)"/>
        <arg name="slam" value="false"/>
    </include>
</launch>
